net/lp_rpc.proto

```diff
@@ -368,6 +376,37 @@ message NotifySegment {
     reserved 33; // Formerly "repeated VideoProfile fullProfiles"
 }
 
+// Sent by the aiworker to register itself to the orchestrator.
+message RegisterAIWorkerRequest {
+
+    // Shared secret for auth
+    string secret = 1;
+
+    // AIWorker capacity
+    int64 capacity = 2;
+
+    // AIWorker capabilities
+    Capabilities capabilities = 3;
+}
+
+// Sent by the orchestrator to the aiworker
+message NotifyAIJob {
+    // URL of the input to use.
+    string url      = 1;
```

Comment:
`url` is the node storage url passed down to the worker to download the input file. I think Nico's version passes all the data including input files in the `data` field.

Reply:
This has been updated and moved to AIJobData in proto file

Reply:
I still see this `url` field, shouldn't we remote it?

Reply:
That is in AIJobData now. https://github.com/livepeer/go-livepeer/blob/dabc0e8db9f95cedbfd4a15de444e44566ba481e/net/lp_rpc.proto#L394

I guess this comment history followed the field since it is the first field?

The requestData is an exact copy of the request received that matches to the ai-worker types. I wanted to keep that field like that so a added the AIJobData shell to hold.

Reply:
@leszko is this resolved?

Reply:
TBH I'd prefer to put this `url` into `requestData`, because AFAIU not every request has url

Reply:
I have moved the `url` to the requestData field.